serial_port: "/dev/ttyUSB0"
baud_rate: 115200